proto/cometbft/abci/v1/types.proto

```diff
@@ -516,8 +515,11 @@ message Validator {
 
 // ValidatorUpdate is a singular update to a validator set.
 message ValidatorUpdate {
-  cometbft.crypto.v1.PublicKey pub_key = 1 [(gogoproto.nullable) = false];
```

Comment:
Should it (at least) deprecate the corresponding message definition?

Reply:
We're using it in other places (Validator, remote signer).

Reply:
That doesn't look right, but doesn't belong to this PR.
Could you please open an issue to decide what to do with all other places that still use `cometbft.crypto.v1.PublicKey`?

Reply:
Or maybe I'm missing something...

Reply:
👍